docs/en/stack/security/get-started-security.asciidoc

```diff
@@ -0,0 +1,311 @@
+
+The list of users should now contain all of the built-in users as well as the 
+two you created. It should also show the appropriate roles for your users:
+
+[role="screenshot"]
+image::security/images/management-users.jpg["User management screenshot in Kibana"]
+
+[float]
+[[gs-logstash-security]]
+=== Add users in {ls}
+
+In order for {ls} to send data successfully to {es}, you must configure its 
+authentication credentials. You must configure credentials separately for each 
+of the {es} plugins in your {ls} configuration file.  For example, update the `demo-metrics-pipeline.conf` file in your {ls} directory:
+
+[source,ruby]
+----
+input {
+  beats {
+    port => 5044
+  }
+}
+
+filter {
+  if [system][process] {
+    if [system][process][cmdline] {
+      grok {
+        match => {
+          "[system][process][cmdline]" => "^%{PATH:[system][process][cmdline_path]}"
+        }
+        remove_field => "[system][process][cmdline]"
+      }
+    }
+  }
+}
+
+output {
+  elasticsearch {
+    hosts => "localhost:9200"
+    manage_template => false
+    index => "%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}"
+    user => metricbeat_internal <1>
+    password => your_password <2>
+  }
+}
+----
+<1> Specify the `metricbeat_internal` user that you created earlier in this tutorial. 
+<2> Specify the password that you chose for this user ID.
+
+<<gs-start-logstash,Start {ls}>>. 
+
+[float]
+[[gs-metricbeat-security]]
+=== Add users in {metricbeat}
+
+If you were connecting directly from {metricbeat} to {es}, you would need to 
+configure authentication credentials for the {es} output in the {metricbeat} 
+configuration file. 
+
+In <<get-started-elastic-stack>>, however, you configured {metricbeat} to send 
+the data to {ls} for additional parsing, so no extra settings are required in 
+{metricbeat}. 
+
+<<gs-start-metricbeat,Start {metricbeat}>>. 
+
+[float]
+[[gs-view-security]]
+=== View system metrics in {kib}
+
+Log in to {kib} with the user ID that has `metricbeat_reader` role (for example, 
+`jdoe`). 
+
+This user should have authority to visualize the system metrics (for example, on 
+the *Discover* page or in the http://localhost:5601/app/kibana#/dashboard/Metricbeat-system-overview[{metricbeat} system overview dashboard]). 
+
+[float]
+[[gs-security-nextsteps]]
+==== What's next?
+
+Congratulations! You've successfully set up authentication and authorization by 
+using the native realm. You learned how to create user IDs and roles that 
+prevent unauthorized access to the {stack}. 
+
+Next, you'll want to try other features that are unlocked by your trial license, 
+such as {ml}. See <<ml-getting-started,Getting started with {ml}>>. 
+
+Later, when you're ready to increase the number of nodes in your cluster or set 
+up an production environment, you'll want to encrypt communications across the 
+{stack}. To learn how, read <<encrypting-communications>>. 
+
```

Comment:
Perhaps mention that the `logstash_system` and `beats_system` user are used for monitoring, and refer to those docs?
We told them to remember those passwords when they ran `elasticsearch-setup-passwords`, but we haven't told them what to do with those users.

Reply:
Good idea, done!